test/interception.jest.js

```diff
@@ -19,7 +19,7 @@ const fs = require('fs');
 const path = require('path');
 const { helper } = require('../lib/helper');
 const vm = require('vm');
-const {FFOX, CHROMIUM, WEBKIT} = require('./utils').testOptions(browserType);
+const {FFOX, CHROMIUM, WEBKIT} = testOptions;
```

Comment:
I wonder how it worked before? HEADLESS was in global somewhere?
```suggestion
const {FFOX, CHROMIUM, WEBKIT, HEADLESS} = testOptions;
```

Reply:
Yes, I think dgozman injected it into the environment